src/cfn/template.yaml

```diff
@@ -111,6 +140,41 @@ Resources:
             Action:
               - sts:AssumeRole
 
+  DynamoDBEndpoint:
```

Comment:
May as well add the S3 one as well.

Reply:
I thought we didn't need S3 from the container?

Reply:
Are we not going to archive the text in S3 at some point using DynamoDb streams? All that traffic will go over the internet otherwise.